From d91fff0fd02af51df345e39cb996aabe57d4902b Mon Sep 17 00:00:00 2001
From: CraigX Kewley <craigx.kewley@intel.com>
Date: Fri, 24 Jun 2016 12:12:57 +0100
Subject: [PATCH 29/98] x86: Platform: BXT: Remove GP MRB board driver.

Board driver was registering i2c amplifier chip. This is no longer
required as we are now using ACPI instead.

Signed-off-by: CraigX Kewley <craigx.kewley@intel.com>

Conflicts:
	arch/x86/platform/bxt/Makefile
---
 arch/x86/platform/bxt/Makefile      |  1 -
 arch/x86/platform/bxt/gpmrb-board.c | 23 -----------------------
 2 files changed, 24 deletions(-)
 delete mode 100644 arch/x86/platform/bxt/gpmrb-board.c

diff --git a/arch/x86/platform/bxt/Makefile b/arch/x86/platform/bxt/Makefile
index 2dc47f6..e69de29 100644
--- a/arch/x86/platform/bxt/Makefile
+++ b/arch/x86/platform/bxt/Makefile
@@ -1 +0,0 @@
-obj-$(CONFIG_GPMRB_BRD) += gpmrb-board.o
diff --git a/arch/x86/platform/bxt/gpmrb-board.c b/arch/x86/platform/bxt/gpmrb-board.c
deleted file mode 100644
index d77dec8..0000000
--- a/arch/x86/platform/bxt/gpmrb-board.c
+++ /dev/null
@@ -1,23 +0,0 @@
-#include <linux/init.h>
-#include <linux/module.h>
-#include <linux/device.h>
-#include <linux/kernel.h>
-#include <linux/i2c.h>
-
-static const struct i2c_board_info i2c_bus_info[] = {
-
- { I2C_BOARD_INFO("tdf8532", 0xD8>>1) },
-
-};
-
-static int __init gpmrb_board_init(void)
-{
-  int ret;
-  printk("Init gpmrb board\n");
-  ret = i2c_register_board_info(4, &i2c_bus_info[0], 1);
-  printk("Init gpmrb board: i2c_register function returned: %d\n",ret);
-  return ret;
-}
-
-arch_initcall(gpmrb_board_init);
-MODULE_LICENSE(GPL);
-- 
1.9.1

